src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ContactPerson from "../ContactPerson";
+import { FooterTitle, PageFooter, PersonContaier } from "./styled";
+
+function Footer(props : any) {
+  var {pageText} = props;
+  return(
+    <PageFooter>
+      <FooterTitle>{pageText}</FooterTitle>
+      <PersonContaier>  
+        <ContactPerson image="https://www.iconninja.com/files/592/204/968/communication-male-cyber-avatar-man-person-crime-human-profile-head-anonymous-hacker-creative-icon.svg" name="Erlend Stenlund" email="erlend.stenlund@itera.com"/>
```

Comment:
Are you sure you want to add yourself as a contact person ?

Reply:
@LERoise I'm definitely not sure about that 😅
Would you like to be a contact person?

Reply:
Do we need any specific contact person? It'd be better to just link to the mad-learning channel on Slack